.eslintrc

```diff
@@ -46,7 +46,7 @@
   "overrides": [
     {
       "files": [
-        "**/src/**/*.js"
```

Comment:
I don't really understand this as an equivalent change. `src` directories were across modules, but it's now only targeting the UI module.